### PR TITLE
Closes #614: Add hello world test for ruby image

### LINKED ISSUE #614: [test] ruby-hello-world
Create a simple hello-world application to ensure that the language can compile (if necessary) and run (see ./test/tests/hylang-sh for an example of using expected-std-out.txt).

### PR DESCRIPTION
**Pull request by MBonell**
Hello world test added!
Closes #614 
Signed-off-by: Marcela Bonell <marcelabonell@gmail.com>

### CODE REVIEW

File: test/config.sh [hunk 1]
```diff
@@ -106,8 +106,9 @@ imageTests+=(
 	'
 	[ruby]='
 		ruby-standard-libs
-		ruby-gems
-		ruby-bundler
+        ruby-gems
+        ruby-bundler
+        ruby-hello-world
```
Comment (yosifkit) [inline test/config.sh]: Just swap these spaces to tabs :wink: and then LGTM.

**Comment by MBonell** [comment]
Done @yosifkit :+1:

**Comment by yosifkit** [comment]
Thanks! :tada: LGTM

**Comment by tianon** [comment]
Nice, LGTM :+1:

**Pull request merged**